windows/vhba_hot_add_remove/set_boot_disk.yml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2022 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Set the boot disk to the specified one.
+# Parameters:
+#   hdd_name: the specified boot disk name. E.g. 'Hard disk 1' which can be got from vSphere Web UI
+#
+- name: Check required parameter
+  ansible.builtin.assert:
+    that:
+      - hdd_name is defined
+      - hdd_name
+    fail_msg: "hdd_name is not defined"
+
+# Shutdown guest OS
```

Comment:
No need in powered off state to set boot order and boot disk.

Reply:
Thanks.